distributed/comm/registry.py

```diff
@@ -60,10 +60,32 @@ def get_local_address_for(self, loc):
 def get_backend(scheme):
     """
     Get the Backend instance for the given *scheme*.
+
+    New backends can be registered using setuptools entry_points, e.g.:
+    setup(..., entry_points={
+            'distributed.comm.backends': [
+                'udp=dask_udp.backend:UDPBackend',
+            ]
+        },
+    )
     """
+
     backend = backends.get(scheme)
     if backend is None:
-        raise ValueError(
-            "unknown address scheme %r (known schemes: %s)" % (scheme, sorted(backends))
+        import pkg_resources
+
+        backend = next(
```

Comment:
Do you know how multiple entrypoints with the same name handled?

Reply:
If two projects register the same name scheme, then the current code will pick the first one as far loaded  dictated by system path.

Reply:
Are you worried at all about "stale entrypoints" being left around when you upgrade a library providing an entrypoint (perhaps while developing?) Perhaps warn if there are unused entrypoints? I may be being paranoid here.

Reply:
I am not at the moment. In the future, we could provide a simple script to list all the entry points registered for dask. For now, I don't think it is an issue.

Reply:
Sounds good to me.